.gitlab-ci.yml

```diff
@@ -73,4 +78,20 @@ release:
     - poetry config pypi-token.pypi $PYPI_PASSWORD
     - make publish
     - shasum -a 256 dist/* > dist/checksums.txt
+
+
+release-github:
+  stage: release
+  artifacts:
```

Comment:
Is this needed? Isn't it for adding artifacts, which is only needed to be done in the step calling `make build`?

Reply:
Removed